ansible/configs/sap-hana/pre_software.yml

```diff
@@ -17,7 +17,17 @@
   become: true
   gather_facts: False
   tasks:
-    - command: subscription-manager release --set=8.1
+    - name: Lock RHEL release to SAP supported one
+      redhat_subscription:
+        state: present
+        server_hostname: "{{ set_repositories_satellite_url }}"
+        activationkey: "{{ set_repositories_satellite_activationkey }}"
+        org_id: "{{ set_repositories_satellite_org | default(satellite_org) }}"
+        release: 8.1
+      register: lock_result
+      until: lock_result.rc == 0
```

Comment:
return code ?

Since you're not using `command` anymore, i'd use this instead:

```yaml
until: lock_result is succeeded
```